types/sinch-rtc-browser/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for non-npm package sinch-rtc-browser 2.31
 // Project: https://developers.sinch.com/docs/in-app-calling/js-cloud/
-// Definitions by: Johan Aschan <https://github.com/sinch-johana>
+// Definitions by: Sinch Johana <https://github.com/sinch-johana>
```

Comment:
this person has to give ACK to be listed here, DT sends notifications
to package owners:
https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners